### PR TITLE
Add missing FQN for common resource attributes

### PR DESCRIPTION
**Pull request by cartermp**
fixes https://github.com/open-telemetry/opentelemetry.io/issues/4328

### CODE REVIEW

**Review by zeitlinger — CHANGES_REQUESTED** [review]
this is now deprecated: https://github.com/open-telemetry/semantic-conventions-java/blob/9202fd6cdc1b21e05558d17b6f13608345a3f80b/semconv/src/main/java/io/opentelemetry/semconv/ServiceAttributes.java#L15

**Comment by theletterf** [comment]
Should this be closed then?

File: content/en/docs/languages/java/exporters.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ public class DiceApplication {
 
   @Bean
   public OpenTelemetry openTelemetry() {
-    Resource resource = Resource.getDefault().toBuilder().put(SERVICE_NAME, "dice-server").put(SERVICE_VERSION, "0.1.0").build();
+    Resource resource = Resource.getDefault().toBuilder().put(ResourceAttributes.SERVICE_NAME, "dice-server").put(ResourceAttributes.SERVICE_VERSION, "0.1.0").build();
```
Comment (zeitlinger) [inline content/en/docs/languages/java/exporters.md:134]:
```suggestion
    Resource resource = Resource.getDefault().toBuilder().put(ServiceAttributes.SERVICE_NAME, "dice-server").put(ServiceAttributes.SERVICE_VERSION, "0.1.0").build();
```

**Comment by zeitlinger** [comment]
> Should this be closed then?

I added a suggestion.
The import is still wrong though.

**Comment by zeitlinger** [comment]
@theletterf I now have an alternative suggestion: 

we can move the example code to https://github.com/open-telemetry/opentelemetry-java-examples and then embed the example code using `code_excerpter`.

See https://github.com/open-telemetry/opentelemetry.io/issues/4276

I'm happy to help :smile:

**Comment by theletterf** [comment]
Nice! Could you try that in a draft PR?

**Comment by zeitlinger** [comment]
Superceded by https://github.com/open-telemetry/opentelemetry.io/pull/4642

**Pull request closed without merging**